tests/end2end/playwright/n_to_m_relations.spec.js

```diff
@@ -1,431 +1,438 @@
-        await expect(childNaturalAreasTable.locator("tbody tr").nth(0).locator("td").nth(2)).toHaveText("Étang de la Vignalie");
-
-        // unlink area from birds, this should delete the pivot record
-        page.once('dialog', dialog => {
-            expect(dialog.message()).toBe("Are you sure you want to unlink the selected feature from \"Birds\" layer?");
-            return dialog.accept();
-        });
-
-        let unlinkPivotFeature = page.waitForResponse(response => response.request().method() === 'POST' && response.request().postData()?.includes('%22bird_id%22+%3D+%279%27') === true)
-        await childNaturalAreasTable.locator("tbody tr").nth(0).locator("td").nth(0).locator("lizmap-feature-toolbar").locator(".feature-toolbar button[data-bs-title='Unlink child']").click();
-        await unlinkPivotFeature;
-
-
-        await page.waitForTimeout(300);
-
-        await expect(childNaturalAreasTable.locator("tbody tr")).toHaveCount(1)
-        await expect(childNaturalAreasTable.locator("tbody tr").nth(0).locator("td").nth(0)).toHaveText("No data available in table");
-
-        // delete a bird record, this should remove pivot records too
-        page.once('dialog', dialog => {
-            expect(dialog.message()).toBe("Are you sure you want to delete the selected feature? \n\nThe related links with the following layers:\n \nNatural areas \n\nwill be also deleted");
-            return dialog.accept();
-        });
-
-        let deleteBirdFeature = page.waitForResponse(response => response.request().method() === 'POST'
-            && response.request().postData()?.includes('GetFeature') === true
-            && response.request().postData()?.includes('birds') === true
-            && response.request().postData()?.includes('extent') === true)
-        await expect(birdsTable.locator("tbody tr").nth(6).locator("td").nth(0).locator("lizmap-feature-toolbar").locator(".feature-toolbar button[data-bs-title='Delete']")).toHaveCount(1);
-        await birdsTable.locator("tbody tr").nth(6).locator("td").nth(0).locator("lizmap-feature-toolbar").locator(".feature-toolbar button[data-bs-title='Delete']").click();
-        await deleteBirdFeature;
-
-        await expect(birdsTable.locator("tbody tr")).toHaveCount(8);
-
-        // popup behavior
-        await page.locator("#bottom-dock-window-buttons button.btn-bottomdock-clear").click()
-
-        // click on map to get popup list
-        let getFeatureInfoRequestPromise = page.waitForRequest(request => request.method() === 'POST' && request.postData()?.includes('GetFeatureInfo') === true);
-        await page.locator('#newOlMap').click({
-            position: {
-                x: 413,
-                y: 232
+import {ProjectPage} from "./pages/project";
+
+test.describe('N to M relations',
+    {
+        tag:['@write'],
+    },
+    () => {
+        test('Attribute table and popup behavior', async ({ page }) => {
+            /* eslint-enable no-unused-vars */
```

Comment:
This could be removed, as enabled is the default value.

Feel free to skip, just if you need to re-commit this PR